core/src/apps/cardano/certificates.py

```diff
@@ -187,20 +176,13 @@ def _validate_pool_metadata(pool_metadata: CardanoPoolMetadataType) -> None:
     assert_certificate_cond(all((32 <= ord(c) < 127) for c in pool_metadata.url))
 
 
-def _cborize_pool_owners(
-    keychain: seed.Keychain, pool_owners: list[CardanoPoolOwnerType]
-) -> list[bytes]:
-    result = []
-
-    for pool_owner in pool_owners:
-        if pool_owner.staking_key_path:
-            result.append(get_public_key_hash(keychain, pool_owner.staking_key_path))
-        elif pool_owner.staking_key_hash:
-            result.append(pool_owner.staking_key_hash)
-        else:
-            raise ValueError
-
-    return result
+def cborize_pool_owner(keychain: seed.Keychain, pool_owner: CardanoPoolOwner) -> bytes:
```

Comment:
This one is called from `sign_tx.py`. But maybe it might be better if `_process_certificates`, `_process_pool_owners` and `_process_pool_relays` were moved to `certificates.py` from `sign_tx.py`. I'm not 100% sure though, I like that all the processing is happening in `sign_tx.py`.